compose/ui/ui-uikit/src/uikitMain/kotlin/androidx/compose/ui/utils/Accessibility.uikit.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.uikit.utils
+
+import platform.UIKit.UIView
+
+class AccessibilityElement(
```

Comment:
```suggestion
@InternalComposeUiApi
class AccessibilityElement(
```

Is this public only because we need access it from the native code?

If so, let's mark it with this annotation. It means that this API is used only by our other API inside `ui-*` module family.

Reply:
Sure! The class `AccessibilityElement` is a temp class, added here to verify that solution works, So you can modify or remove it in any way you want.

Reply:
Sure! The class `AccessibilityElement` is a temp class, added here to verify that solution works, So you can modify or remove it in any way you want.